README.md

```diff
@@ -1,64 +1,316 @@
+9. Click on *Apply*. Select the created configuration and click *Run*. Now both the projects will run at a time.
+
+- Now navigate to the home page of the project. For this sample, the standard home page URL is <http://localhost:8080
+
+##### Packaging and deploying to container
+
+If you would like to deploy the sample to Tomcat, you will need to make a couple of changes to the source code in both modules.
+
+1. Open msal-webapp-sample/pom.xml
+    - Under `<name>msal-webapp-sample</name>` add `<packaging>war</packaging>`
+    - Add dependency:
+
+         ```xml
+         <dependency>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-tomcat</artifactId>
+          <scope>provided</scope>
+         </dependency>
+         ```
+
+2. Open msal-webapp-sample/src/main/java/com.microsoft.azure.msalwebappsample/MsalWebAppSampleApplication
+
+    - Delete all source code and replace with
+
+    ```Java
+        package com.microsoft.azure.msalwebappsample;
+
+        import org.springframework.boot.SpringApplication;
+        import org.springframework.boot.autoconfigure.SpringBootApplication;
+        import org.springframework.boot.builder.SpringApplicationBuilder;
+        import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+        @SpringBootApplication
+        public class MsalWebSampleApplication extends SpringBootServletInitializer {
+         public static void main(String[] args) {
+          SpringApplication.run(MsalWebSampleApplication.class, args);
+         }
+
+         @Override
+         protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+          return builder.sources(MsalWebSampleApplication.class);
+         }
+        }
+    ```
+
+3. Open a command prompt, go to the root folder of the project, and run `mvn package`
+
+- This will generate a `msal-webapi-sample-0.1.0.war` file in your /targets directory.
+- Rename this file to `ROOT.war`
+- Deploy this war file using Tomcat or any other J2EE container solution.
+- To deploy on Tomcat container, copy the .war file to the webapis folder under your Tomcat installation and then start the Tomcat server.
+
+This WAR will automatically be hosted at `http:<yourserverhost>:<yourserverport>/`
+
+Tomcats default port is 8080. This can be changed by
+    - Going to tomcat/conf/server.xml
+    - Search "Connector Port"
+    - Replace "8080" with your desired port number
+
+Example: `http://localhost:8080/msal4jsample`
+
+### You're done, run the code
+
+Click on "Login" to start the process of logging in. Once logged in, you'll see the account information for the user that is logged in and a Button "Call OBO API" , which will call the Microsoft Graph API with the OBO token and display the basic information of the signed-in user. You'll then have the option to "Sign out".
+
+## About the Code
```

Comment:
There should be a section about the `/.default` scope, which is the responsible for the OBO to work.

Reply:
Added